.travis.yml

```diff
@@ -1,14 +1,14 @@
 sudo: false
 language: python
 python:
-  - "3.5"
-  - "3.6"
+- '3.5'
+- '3.8'
 branches:
   only:
-    - master
+  - master
```

Comment:
I was going to complain about the indentation change, but it looks like most of the examples in the Travis CI docs use the updated indentation style, so I suppose we should keep it this way.